### DISCUSSION TITLE
Signpost, life counter and monitor graphics need updating?

### DISCUSSION THREAD

**Discussion by rocketsailor** [Techokami/SonicWorldsNext#70] [Q&A] [answered]
Amy's signpost and life icon have not been changed since she was added. Is that planned for a future update (before SAGE 23)?

I would be happy to make new ones with the palette from her sprite sheet if somebody else isn't already doing so.

**Comment by Renhoex** [comment, marked as answer]
We opted to stick with the original worlds sprites in some places as on a personal preference, we did experiment with using the origins sprites but the team thought the original worlds artwork looked better and it's an incentive to insert the developers own artwork.

It's also the reason why the player sprites are using the sonic 3 prototype style instead of sonic 1, 2 or 3 so that if someone wants a specific style the work's not already done for them and they can get their own identity for their fangames.

**Comment by rocketsailor** [comment]
Okay. Thank you for the explanation!
